src/coreclr/jit/lsraarmarch.cpp

```diff
@@ -619,6 +619,12 @@ int LinearScan::BuildBlockStore(GenTreeBlk* blkNode)
         switch (blkNode->gtBlkOpKind)
         {
             case GenTreeBlk::BlkOpKindUnroll:
+#ifdef TARGET_ARM64
+                if (size > 2 * FP_REGSIZE_BYTES)
+                {
+                    buildInternalFloatRegisterDefForNode(blkNode, internalFloatRegCandidates());
```

Comment:
I guess you request temporary registers conservatively, that is, the temporary registers might not actually get used depending on the unrolling strategy actually used?